### PR TITLE
🍉Vishal

### PR DESCRIPTION
**Pull request by vishalsingh2972**
Adding my name to the contributor list !

### CODE REVIEW

File: .idea/vcs.xml
Comment (alisolanki) [inline .idea/vcs.xml:1]: delete this file

File: .idea/modules.xml
Comment (alisolanki) [inline .idea/modules.xml:1]: delete this file

File: .idea/misc.xml
Comment (alisolanki) [inline .idea/misc.xml:1]: delete this file

File: .idea/.gitignore
Comment (alisolanki) [inline .idea/.gitignore:1]: make sure to not change this file

**Review by alisolanki — CHANGES_REQUESTED** [review]

**Comment by vishalsingh2972** [comment]
@alisolanki cannot make changes as deleted the previous fork, creating new PR....please accept that

File: .idea/Welcome-to-Open-Source.iml
[the remark this answers is not in the document]
Comment (vishalsingh2972) [reply .idea/Welcome-to-Open-Source.iml:1]: unable to commit if deleting this so kept this deleted the rest, please accept the new PR!

**Pull request closed without merging**